src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -83,11 +83,12 @@ public class Session {
 
             var useridentity = User as WindowsIdentity;
             // In remote broker User Identity type is always WindowsIdentity
-            string suppressUI = (useridentity == null) ? string.Empty : "--suppress-ui ";
+            string suppressUI = (useridentity == null) ? string.Empty : "--rhost-suppress-ui ";
+            string isRepl = Id.StartsWithIgnoreCase("REPL") ? string.Empty : "--rhost-interactive ";
```

Comment:
Actually, come to think of it, when the session is created, the creator can specify the arbitrary command line parameters. So let's do that all on the broker client side instead. This shouldn't care about anything except for log verbosity, and that only because it comes from global broker settings.